.github/workflows/ci.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2019, windows-2022]
```

Comment:
Wile I would love to only test on the latest Windows release, there are people that are stuck on older releases. Unless a particular version of Windows is EoL, I would like to still test on those. We can remove 2019, but I would like to keep 2022 and 2025 (which is probably covered by `latest`).